test/e2e/common_suite_test.go

```diff
@@ -291,7 +291,7 @@ func comparePodLogString(ctx context.Context, client klient.Client, customPod v1
 		return podLogString, err
 	}
 
-	if podLogString != expectedPodlogString {
+	if !strings.Contains(podLogString, expectedPodlogString) {
 		return podLogString, errors.New("Error: Pod Log doesn't match with Expected String")
```

Comment:
I wonder if with this change to `contains` the error should be more explicit and mention this?

Reply:
actually we are comparing the string with bunch of other environment variables as well
so implemented with strings.Contains

Reply:
Sorry if I wasn't clear. What I means was changing the error message to something like `"Error: Pod Log doesn't contain Expected String"` rather than match.